frontend/src/slices/chats/actions.ts

```diff
@@ -59,4 +60,29 @@ const createMessage = createAsyncThunk<
   return await chatMessagesApi.createChatMessage(payload);
 });
 
-export { createChat, createMessage, getAllChats, getCurrentChatMessages };
+const deleteChat = createAsyncThunk<
+  ChatGetAllItemResponseDto[],
+  number,
+  AsyncThunkConfig
+>(`${sliceName}/delete-chat`, async (id, { extra, getState, dispatch }) => {
+  const { chatApi } = extra;
+  await chatApi.deleteChat(id);
+
+  const {
+    chats: { chats },
+  } = getState();
+
+  dispatch(appActions.navigate(AppRoute.CHATS));
+
+  return chats.filter((chat) => {
```

Comment:
this should be done inside reducer

Reply:
Could you, please, check this comment by @v0ldemar01 https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/327#discussion_r1335150470

Reply:
@OlenaYatseniuk please follow @Liza-Veis comment, I have mistaken with it

Reply:
Ok, I'll remove it to slice